packages/odyssey-react-mui/src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,79 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { AlertColor } from "@mui/material";
+import React, { useState } from "react";
+import { Alert, Link } from "../..";
+
+export interface BannerProps {
+  /**
+   * Determine the color and icon of the alert
+   */
+  severity?: AlertColor;
+  /**
+   * Sets the ARIA role of the alert
+   * ("status" for something that dynamically updates, null for something
+   * unchanging)
+   */
+  role?: "status" | undefined;
```

Comment:
`role` is an HTML term, so I wouldn't recommend using it for something other than accessibility roles.

Reply:
MUI uses this prop to set the HTML attribute under the hood.

> role:	The ARIA role attribute of the element.

(from https://mui.com/material-ui/api/alert/)